### DISCUSSION TITLE
What version of IntelliJ IDEA to use for working on this project?

### DISCUSSION THREAD

**Discussion by Kshitij09** [dropbox/Store#421] [Q&A] [answered]
I want to browse the Store source code and hence I cloned the repository and tried opening the project in IntelliJ IDEA. My machine specifications are as follows
```
IntelliJ IDEA 2021.3.3 (Community Edition)
Build #IC-213.7172.25, built on March 15, 2022
Runtime version: 11.0.14.1+1-b1751.46 aarch64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
macOS 12.2.1
GC: G1 Young Generation, G1 Old Generation
Memory: 2048M
Cores: 10

Kotlin: 213-1.5.10-release-949-IJ7172.25
```
I think I'm using a compatible IDEA version but still, I'm getting the following error
```
This version of the Android Support plugin for IntelliJ IDEA (or Android Studio) cannot open this project, please retry with version 2021.1.1 or newer.
```

Any help in IDE setup, both IntelliJ and Android Studio would be much appreciated. Thanks!

**Comment by matt-ramotar** [comment, marked as answer]
Hey! Hoping you solved this already but in case not and for future readers - I am on IntelliJ IDEA 2022.2 (Ultimate Edition). You can download the latest here https://www.jetbrains.com/idea/download
